app/decorators/physical_chassis_decorator.rb

```diff
@@ -0,0 +1,39 @@
+class PhysicalChassisDecorator < MiqDecorator
+  def self.fonticon
+    'pficon pficon-cluster'
+  end
+
+  def fileicon
+    "svg/vendor-#{vendor.downcase}.svg"
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => physical_servers.count,
+        :tooltip => _('Servers')
+      },
+      :top_right    => {
+        :fileicon => fileicon,
```

Comment:
Hmm, this seems as redundancy to me. If you don't know what else to display here, leave rather it empty.